### ISSUE TITLE
[8] Capture Github Statistics for each manifest we collect

### ISSUE THREAD

**Issue by SaketJNU** [openshiftio/openshift.io#983]
### Description:
Collect Github Statistics for Vertx and Springboot Github Manifest files

#### Acceptance Criteria:
Each Vertx and Springboot manifest files should also capture Github Statistics

##### Task:
- [x] Finalize the json structure.
- [x] Create an parser for extracting github-statistics from repos.
- [x] Integrate the parser with manifest-data collector, so that during dependency snapshot collection we can also collect github-statistics.
- [x] Modify jobs to capture Github Statistics.
- [x] Execute the job and collect the required data at S3.
- [x] Create a final list as per finalized json structure.
#### Dependency 
Once the PR  fabric8-analytics/fabric8-analytics-jobs#171 will get merged into master and will get deployed on prod, owners of repo. like @miteshvp can run the query like ```filename:pom.xml vertx language:java stars:>1``` and ```filename:pom.xml springboot language:java stars:>1``` and can get the desired data.

**Comment by joshuawilson** [comment]
This looks more like a `user-story` then a `task`, it does list tasks.

**Comment by miteshvp** [comment]
@joshuawilson - this is a task and this effects the data collection part to be modified to capture more

**Comment by joshuawilson** [comment]
Just saying, it looks like a user-story to me. But it is your issue so you can call it what you want. :)

**Comment by SaketJNU** [comment]
A PR https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/351 related to 2, 3, and 4th tasks from top is opened. Once it will get approved, one more PR is required for the last task in order to complete this issue.

**Comment by SaketJNU** [comment]
A PR https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/171  related to last task from top is opened. When both PR's will get approved, merged and deployed on prod, then only remaining task (5th task) is : run the query on swagger, get data and make @sara-02 unblock from data-side in new structure of PGM.

**Comment by msrb** [comment]
Just curious, why do we need this data?

**Comment by SaketJNU** [comment]
@msrb as updated version of PGM needs some more data as mentioned by @sara-02 in the format as given in https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis/pull/61. This data will make the building block for new logic of PGM when the packages do't have tags, then other data-values my help to find-out companion, alternate, outlier packages and many-more. I not aware of exact logic "how they will do it" , but this will help in needful. 
Thanks

**Comment by SaketJNU** [comment]
A PR https://github.com/openshiftio/saas-analytics/pull/107 is opened

**Comment by SaketJNU** [comment]
Now the desired list is available in "prod-manifests-new" s3 bucket. 
So, We can close this issue.
